packages/affix/index.js

```diff
@@ -1,33 +1,30 @@
 import { html } from 'lit';
 
-import { suffix, prefix } from '@warp-ds/css/component-classes';
+import { classNames } from '@chbphone55/classnames';
+import { suffix as ccSuffix, prefix as ccPrefix } from '@warp-ds/css/component-classes';
 import WarpElement from '@warp-ds/elements-core';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import '@warp-ds/icons/elements/search-16';
 import '@warp-ds/icons/elements/close-16';
 
-import { fclasses } from '../utils';
-
-class WarpAffix extends WarpElement {
-  static styles = [WarpElement.styles];
+import { kebabCaseAttributes } from '../utils';
 
+class WarpAffix extends kebabCaseAttributes(WarpElement) {
   static properties = {
     ariaLabel: { type: String, attribute: 'aria-label' },
```

Comment:
If we use the `kebabCaseAttributes()` wrapper, the `attribute: 'aria-label'` option is redundant.

Reply:
Updated in this commit: [5459cf7](https://github.com/warp-ds/elements/pull/174/commits/5459cf7eeceee45bb633efda91311217978ae8df)